### PR TITLE
Improve instructions on how to build from source

### PR DESCRIPTION
**Pull request by ktec**
This commit refactors the instructions on how to create a project
using github source to make it more explicit and easier to follow for
new comers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,21 +17,27 @@ API documentation is available at [https://hexdocs.pm/phoenix](https://hexdocs.p
 
 We appreciate any contribution to Phoenix. Check our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [CONTRIBUTING.md](CONTRIBUTING.md) guides for more information. We usually keep a list of features and bugs [in the issue tracker][4].
 
-### Running a Phoenix master project
+### Create a new Phoenix project from source
+
+In order to create a new project using the latest Phoenix source installer (the `phoenix.new` Mix task) you will need to ensure two things.
+
+1. Remove any previously installed `phoenix_new` archives so that Mix will pick up the local source code. This can be done with `mix archive.uninstall phoenix_new-#.#.#.ez` or by simply deleting the file, which is usually in `~/.mix/archives/`.
```
Comment (nicholaswyoung) [inline README.md]: 👍 This frustrated me the first time around, and really should be included.

**Comment by chrismccord** [comment]
<3 <3 <3

File: README.md (hunk 1 above)
Comment (ericmj) [inline README.md]: It should be just `mix archive.uninstall phoenix_new.ez`

Comment (chrismccord) [inline README.md]: 👍  done

**Comment by ktec** [comment]
👍

**Pull request merged**